www/src/templates/component-page-template.tsx

```diff
@@ -136,7 +138,10 @@ export default function PageTemplate({
             <p className="small mb-0">{mdx.frontmatter.notes}</p>
           </Alert>
         )}
-        <h1 className="mb-4">{mdx.frontmatter.title}</h1>
+        <div className="d-flex justify-content-between">
+          <h1 className="mb-4">{mdx.frontmatter.title}</h1>
+          <LeaveFeedback as={Hyperlink} />
```

Comment:
Per the [issue](https://github.com/openedx/paragon/issues/2373), I believe the intent for "Leave feedback" now is to be a small `Button` (`variant="outline-primary"`) that acts as a `Hyperlink` here in `component-page-template` and below the page content (though not the footer).